### PR TITLE
poll detail component

### PR DESCRIPTION
**Pull request by xi**
this implements a react component and styling for the poll detail view. It currently uses hardcoded sample data as the API and models are not ready yet.

### CODE REVIEW

File: apps/polls/assets/Polls.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+              let percent = Math.round(option.count / total * 100)
+              let highlight = option.count === max
+
+              if (this.state.showResult || !this.state.active) {
+                return (
+                  <div className="poll-row" key={i}>
+                    <div className={'poll-row__bar' + (highlight ? ' poll-row__bar--highlight' : '')} style={{width: percent + '%'}} />
+                    <div className="poll-row__number">{ percent }%</div>
+                    <div className="poll-row__label">{ option.label }</div>
+                    &nbsp;
```
Comment (vellip) [inline apps/polls/assets/Polls.jsx]: I don't think it's necessary here, looks fine without.

File: apps/polls/assets/Polls.jsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+              } else {
+                return (
+                  <label className="poll-row" key={i}>
+                    <input
+                      className="poll-row__radio"
+                      type="radio"
+                      name="poll"
+                      value={i}
+                      defaultChecked={checked} />
+                    &nbsp;
```
Comment (vellip) [inline apps/polls/assets/Polls.jsx]: I'd prefer a css solution here. You have the `poll-row__radio` class already there.

File: apps/polls/assets/Polls.jsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        label: 'Ja',
+        count: 22434
+      }, {
+        label: 'Nein',
+        count: 40062
+      }, {
+        label: 'Vielleicht',
+        count: 17627
+      }],
+      own: null,
```
Comment (vellip) [inline apps/polls/assets/Polls.jsx]: Can we name this more explicitly like `ownVote` or something like that?

File: apps/polls/assets/Polls.jsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+                    &nbsp;
+                    { option.label }
+                  </label>
+                )
+              }
+            })
+          }
+        </div>
+
+        { !this.state.active ? (
```
Comment (xi) [inline apps/polls/assets/Polls.jsx]: I spend alot of time fighting with JSX here. Can you tell me how to do it?

**Comment by xi** [comment]
> However, this does not yet work and wont work as implemented for multiple questions in one poll. Can you still add that?

I was not sure whether we want multiple polls in one react component or just multiple react components on one page. This decision might be influenced by the backend API. Whichever we land on, both should be easy to add later on.

File: apps/polls/assets/Polls.jsx (hunk 4 above)
Comment (vellip) [inline apps/polls/assets/Polls.jsx]: Maybe check this for reference: https://facebook.github.io/react/docs/conditional-rendering.html I personally prefer using what they call "Element variables" on that page.

**Pull request merged**